libraries/SITL/SIM_JEDEC.cpp

```diff
@@ -0,0 +1,192 @@
+#include "SIM_JEDEC.h"
+
+#include <errno.h>
+#include <unistd.h>
+
+#include <AP_HAL_SITL/AP_HAL_SITL.h>
+
+using namespace SITL;
+
+extern HAL_SITL& hal;
+
+void JEDEC::open_storage_fd()
+{
+    if (storage_fd != -1) {
+        AP_HAL::panic("Should not have been called");
+    }
+    
+    const char *filepath = filename();
+    uint32_t flags = O_RDWR|O_CLOEXEC;
+    storage_fd = open(filepath, flags, 0777);
```

Comment:
```suggestion
    storage_fd = open(filepath, flags, 0660);
```

... similarly elsewhere, and apologies if this is just copy/pasted :-)

Reply:
Done!